src/_assets/css/main.scss

```diff
@@ -752,21 +752,26 @@ main .content {
   h5,
   h6 {
     .anchor {
-      box-sizing: border-box;
       float: left;
       margin-left: -($icon-size + $anchor-padding);
-      max-width: $icon-size;
-      overflow: hidden;
       padding-right: $anchor-padding;
+      overflow: hidden;
 
       .octicon-link {
         @include transition(.1s);
 
-        background-image: asset_url('ic_link_black_24px.svg');
-        display: block;
-        height: $icon-size;
+        @include md-icon-before("link");
+        font-size: 24px;
+        color: $site-color-body;
         opacity: 0;
-        width: $icon-size;
+
+        &:hover {
+          color: lighten($site-color-body, 25%);
```

Comment:
Nice touch! What do you think of making the hover color match the one we use for link text?

To me, graying the icon makes it look like it's inactive. Also, some headings with similar UIs copy the link URL, instead of going there, so the blue is perhaps a little clearer. Related issue: #920

Reply:
I like this idea, thanks. Do you think I should just change the hover state to the same one as links or should the base state use the link color as well?

Reply:
I hadn't thought about that... It might be less distracting to keep the base state color the dark gray. But I can see it both ways. One question is whether having the icon blue in base state would look better or worse when the icon is misbehaving. (Iirc, it's partially cut off sometimes.)

Reply:
That makes sense, I think I'll stick with the darker grey for the base state.

As for when it's cut off, I wanted to fix that as well, either providing more space or preferably moving it to the other end of the header(like [web.dev](https://web.dev/minify-css/#loading-unminified-css) has theirs). It's not as straightforward as it seems jekyll-toc generates the anchor without allowing much configuration but I'll likely tackle it in the future.

Reply:
Okay, I've switched the hover color to match link text when hovered. Let me know if that fits what you're thinking. Thanks!

<img width="313" alt="A showcase that the anchor icon is a light blue when hovered" src="https://user-images.githubusercontent.com/18372958/130709660-abe3fac7-a142-4f73-a61d-b5fc3a8255e2.png">